Documentation update

AakashV opened the pull request:
Added Some documentation on, beam current normalisation; added a dm3
file reader, some documentation on peak finding methods

AakashV committed:
Documentation update

Added Some documentation on, beam current normalisation; added a dm3
file reader, some documentation on peak finding methods

msarahan commented on Absolute_Integrator/file_readers/dm3lib_v099.py:
```diff
@@ -0,0 +1,638 @@
+## based on the DM3_Reader plug-in (v 1.3.4) for ImageJ by Greg Jefferis <jefferis@stanford.edu>
+## http://rsb.info.nih.gov/ij/plugins/DM3_Reader.html
+## --
+## Python adaptation: Pierre-Ivan Raynal <raynal@med.univ-tours.fr>
```
Have we asked Pierre-Ivan for permission to include this?

My personal opinion is that this perhaps belongs in another module.  Keep AI as focused as possible on a few things - Image IO can become its own project pretty quickly.

msarahan commented on Absolute_Integrator/file_readers/dm3lib_v099.py:
```diff
@@ -0,0 +1,638 @@
+
+__all__ = ["DM3","version"]
+
+version='0.99beta'
```
Generally, files themselves are not versioned, but rather inherit the version of the project they're in.  I would also remove the v099 from the filename.

AakashV committed:
created a list of functions in normalisation_methods

created groups of where some of the methods performed on the detector
should go.

Deleted dm3lib_v099.py as a better module for opening files can be used
